dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetectorTest.kt

```diff
@@ -70,6 +70,83 @@ internal class GitRepositoryDetectorTest {
             .containsExactlyInAnyOrder(*fakeTrackedFiles.toTypedArray())
     }
 
+    @Test
+    fun `𝕄 remove user information from remote URL 𝕎 detectRepository() { https format }`(
+        @StringForgery(regex = "https://[a-z]+:[a-z]+@github\\.com/[a-z]+/repository\\.git")
+        fakeRemoteUrlWithCredentials: String
+    ) {
+        // Given
+        initializeGit(fakeRemoteUrlWithCredentials)
+
+        // When
+        val result = testedDetector.detectRepositories(fakeProject, fakeSourceSetFolders)
+
+        // Then
+        assertThat(result).hasSize(1)
+        val repository = result.first()
+        assertThat(repository.url).matches("https://@github.com/(.+)/repository.git")
```

Comment:
shouldn't we drop `@` symbol as well after sanitization? is it a correct link which will be working properly with that symbol left?

Reply:
actually I am going to change the way I am sanitizing that URL as I don't like it now. I will just rebuild the URL by removing the `userInformation`